frontend/src/libs/packages/notification/notification.package.ts

```diff
@@ -0,0 +1,32 @@
+import { toast } from 'react-toastify';
+
+import { type NotificatinOptions } from './libs/types/notificatin-options.type.js';
```

Comment:
1. You have typo
2. If this type is used only here to add type for parameters - then maybe let's have it defined in this file locally

Reply:
Was not resolved